corehq/apps/app_manager/views.py

```diff
@@ -1060,6 +1060,10 @@ def qualified_form_name(form):
         context.update(get_app_view_context(request, app))
     else:
         from corehq.apps.dashboard.views import NewUserDashboardView
+        from corehq.apps.style.utils import set_bootstrap_version3
+        from crispy_forms.utils import set_template_pack
+        set_bootstrap_version3()
+        set_template_pack('bootstrap3')
```

Comment:
I took this approach because NewUserDashboardView.dispatch is never called. It only sets the template. If there is a cleaner way to do this, please shout.